README.md

```diff
@@ -93,18 +93,18 @@ On __macOS__, install SDL2 via [Homebrew](http://brew.sh) like so:\
 
 On __Windows__,
 1. Install mingw-w64 from [Mingw-builds](http://mingw-w64.org/doku.php/download/mingw-builds)
-    * Version: latest (at time of writing 6.3.0)
+    * Version: latest (at time of writing 8.1.0)
     * Architecture: x86_64
     * Threads: win32
     * Exception: seh
-    * Build revision: 1
+    * Build revision: 0
     * Destination Folder: Select a folder that your Windows user owns
-2. Install SDL2 http://libsdl.org/download-2.0.php
+2. Download `SDL2-devel-2.0.10-mingw.tar.gz` from http://libsdl.org/download-2.0.php
     * Extract the SDL2 folder from the archive using a tool like [7zip](http://7-zip.org)
-    * Inside the folder, copy the `i686-w64-mingw32` and/or `x86_64-w64-mingw32` depending on the architecture you chose into your mingw-w64 folder e.g. `C:\Program Files\mingw-w64\x86_64-6.3.0-win32-seh-rt_v5-rev1\mingw64`
+    * Copy contents of the `i686-w64-mingw32` and/or `x86_64-w64-mingw32` depending on the architecture you chose into your mingw-w64 installation folder e.g. `C:\Program Files\mingw-w64\x86_64-8.1.0-win32-seh-rt_v6-rev0\mingw64` and `C:\Program Files\mingw-w64\x86_64-8.1.0-win32-seh-rt_v6-rev0\mingw64\x86_64-w64-mingw32`
```

Comment:
Hi @divmgl, is there`i686-w64-mingw32` inside `C:\Program Files\mingw-w64\x86_64-8.1.0-win32-seh-rt_v6-rev0\mingw64` here? At least, that's what I remembered.